linux-tkg-config/prepare

```diff
@@ -6,6 +6,9 @@ _current_kernels=("6.6" "6.5" "6.4" "6.1" "5.15")
 # List of kernels that are no longer maintained either upstream or locally
 _eol_kernels=("6.3" "6.2" "6.0" "5.19" "5.18" "5.17" "5.16" "5.14" "5.13" "5.12" "5.11" "5.9" "5.8" "5.7" "5.4.230" "5.10.135")
 
+# Current CPU scheduler in vanilla Linux kernel
+_default_cpu_sched="eevdf"
```

Comment:
Great thanks! Sorry for thinking about it just now: 

it's best to define `_default_cpu_sched` at the end of `_set_kver_internal_vars()`: this function is called right after the kernel is chosen, to define some internal vars. And we can use `_kver` to know which scheduler is the default:

```bash

if [ $_kver -le 605 ]; then
  # chosen kernel <= 6.5
  _default_cpu_sched="cfs"
else
  # chosen kernel >= 6.6
  _default_cpu_sched="eevdf"
fi

 echo -e "_basekernel='$_basekernel'\n_basever='$_basever'\n_sub='$_sub'\n_kver='$_kver'\n_default_cpu_sched='$_default_cpu_sched'" > "$_where"/BIG_UGLY_FROGMINER
```